h2o-algos/src/main/java/hex/aggregator/AggregatorModel.java

```diff
@@ -132,7 +132,7 @@ public void map(Chunk c1, NewChunk nc) {
           nc.addNum(ArrayUtils.find(keep, gid));
         }
       }
-    }.doAll(exAssignment)._fr.vec(0);
+    }.doAll(Vec.T_NUM,exAssignment).outputFrame().vec(0);
```

Comment:
How come the test didn't catch that? Do we need to adjust the test then as well?

Reply:
The frame was getting populated by the wrong frame. The content populated were wrong but the length of frame etc were as expected. So, the tests do not need to change.

Reply:
Right, the test is not failing. But can we change the test to make sure it catches this type of a bug?

Imagine I would just accidentally overwrite your change. The tests should detect that.

Reply:
Added a new unit test to check the content of the frame.

Reply:
@nikhilshekhar thank you!